app/home/routes.py

```diff
@@ -74,16 +76,17 @@ def create_property():
 
     if form.validate_on_submit():
         img_files = request.files.getlist("prop_photos")
-        temp_dir, temp_file_list = save_images_to_temp_folder(img_files)
+        redis_img_key = save_to_redis(img_files)
 
-        images_list = property_image_handler(temp_dir, temp_file_list)
-        img_list_to_json = json.dumps(images_list)
+        s3_upload_dir = generate_dir_name()
+        output_filenames = resize_images(redis_img_key, s3_upload_dir)
```

Comment:
I  hope this gets only  items from index  1 from the list
leaving out  the folder name at index 0
the  templates could be updated to get folder name from  a variable or  similar